index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./index.css">
+</head>
+<body class="m0">
+    <div class="header-container">
+        <header>
+            <nav class="d-flex justify-between flex-nowrap">
+                <a id="logo" href="#">ArB</a>
+                <img src="./images/menu-icon.png" alt="menu">
+            </nav>
+        </header>
+        <section class="headline d-flex flex-column">
+            <div class="title d-flex flex-column">
+                <h1 class="m0">I’m Abiola <br />
+                    <span>Glad to see you!</span>
+                </h1>
+            </div>
+            <div>
+                <p>
+                    I'm a full-stack developer! I can help you build an interactive 
+                    and fully responsive website. My designs are top notches. A typical design
+                    of my website is the one you are viewing right now. Connect with me
+                    on any of the social media listed below if you would like to see more of 
+                    my work.
+                </p> 
+            </div>
+
+            <div class="connect">
+                <div id="title">LET’S CONNECT</div>
+                <ul class="social d-flex justify-start">
+                   <li><a href="#"><img src="./images/twitter-icon.png" alt="twitter-icon"><a/> </li>
+                   <li><a href="#"><img src="./images/linkedin-icon.png" alt="linkedin-icon"><a/> </li>
+                   <li><a href="#"><img src="./images/medium-icon.png" alt="medium-icon"><a/> </li>
+                   <li><a href="#"><img src="./images/github-icon.png" alt="github-icon"><a/> </li>
+                   <li><a href="#"><img src="./images/hand-icon.png" alt="hand-icon"><a/> </li>
+                </ul>
```

Comment:
- [ Optional ] Good job using a `ul` tag in this part of your design, however, I noticed that the `href` attributes of the links are empty. I would like to suggest that you add your social media URLs like `twitter`, `linkedin` and `github` these links are useful especially if a recruiter would like to reach out to you.

Reply:
Is it okay to a remove link that I don't have the account?

Reply:
Since we want to emulate the figma design as much as possible, you can leave the icons/images of those ones and point their hrefs to the home page.